roles/setup/hetzner_install_image/tasks/main.yml

```diff
@@ -11,7 +11,7 @@
     reboot_timeout: 7200
```

Comment:
overkill number?

Reply:
I agree. It seemed that reboot was never working (at least for me). This high number was the result of trying.

Reply:
Did the reboot work in the end? Feel free to suggest a different value or to resort to the default if you think this makes sense.